lib/namae/parser.y

```diff
@@ -246,13 +259,21 @@ require 'strscan'
     input.rest.strip.split(/\s+/)[0] =~ suffix
   end
 
+  def pronoun?
```

Comment:
Is this actually used anywhere?

Reply:
i just copied it from the suffix implementation.